.ci/project-dependencies.yaml

```diff
@@ -157,3 +157,16 @@ dependencies:
       - project: kiegroup/appformer
       - project: kiegroup/kie-uberfire-extensions
       - project: kiegroup/optaplanner-wb
+
+  - project: kiegroup/process-migration-service
+    dependencies:
+      - project: kiegroup/droolsjbpm-integration
```

Comment:
is this true?  @ruromero what repo PIM needs to be buildable? I think it was in D-I repo only for not fitting anywhere else. but i think it requires only jbpm repo. Please correct me if I am wrong.

Reply:
It only depends on jbpm repo IIRC

Reply:
we got information from https://github.com/kiegroup/process-migration-service/blob/b3ce1e28095fd5c7cd12e57c350da00be7a3ec2e/pom.xml#L8

Reply:
as far as I know, PIM relies on `droolsjbpm-integration` repo as per the pom. I don't think there is any direct dependency on the jbpm module, but I'm might be wrong since I'm pretty new to PIM :)

Reply:
ok, I thought the PIM already is transitioned out of Thorntail, thanks guys for confirmation.